lib/helpers/conferences.rb

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+
```

Comment:
I need to add this otherwise I got

```
# make
rm -rf output
bundle exec nanoc
Loading site... 
Captain! We've been hit!

Message:

SyntaxError: lib/helpers/conferences.rb:24: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:24: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:24: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:31: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:31: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:35: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:35: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:35: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:46: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:46: invalid multibyte char (US-ASCII)
lib/helpers/conferences.rb:46: invalid multibyte char (US-ASCII)

Stack trace:

  0. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/base/entities/code_snippet.rb:35:in `eval'
  1. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/base/entities/code_snippet.rb:35:in `load'
  2. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/base/repos/site_loader.rb:24:in `each'
  3. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/base/repos/site_loader.rb:24:in `site_from_config'
  4. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/base/repos/site_loader.rb:12:in `new_from_cwd'
  5. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/cli/command_runner.rb:24:in `site'
  6. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/cli/command_runner.rb:51:in `load_site'
  7. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/cli/commands/compile.rb:405:in `run'
  8. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/cli/command_runner.rb:12:in `block in call'
  9. /usr/local/bundle/gems/nanoc-4.6.3/lib/nanoc/cli/error_handler.rb:64:in `handle_while'
  ... 30 more lines omitted. See full crash log for details.

If you believe this is a bug in Nanoc, please do report it at
-> https://github.com/nanoc/nanoc/issues/new <-

A detailed crash log has been written to ./crash.log.
Makefile:2: recipe for target 'compile' failed
```